### PR TITLE
chore: make text consistent with new SDK version

### PR DESCRIPTION
**Pull request by ntn-x2**
The text was referencing stuff that was happening in the code snippets, which has largely changed between 0.28 and 0.29. This PR fixes that, and can be considered to be the last one pending before releasing 0.29 out.

It builds on top of https://github.com/KILTprotocol/docs/pull/168.

### CODE REVIEW

**Comment by ntn-x2** [comment]
@rflechtner the changes are rather minor, but I would like you to check the dapp docs section, as I am not familiar with those and I am not 100% sure the flow described there is up to date with the upcoming SDK version.

@Dudleyneedham could you do the same for the domain linkage credential section? It talks about credentials and presentations in a way that is not clear, and would like your feedback on that. Thanks!

File: code_examples/core_features/getting_started/06_verify_attestation.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ export async function main(
   const encodedAttestationInfo = await api.query.attestation.attestations(
     credential.rootHash
   )
-  // This function will throw if encodedAttestationInfo is None
+  // This function will throw if encodedAttestationInfo does not exist.
```
Comment (rflechtner) [inline code_examples/core_features/getting_started/06_verify_attestation.ts]: that's a little ambiguous, should we say if the attestation does not exist?

File: docs/develop/01_sdk/02_cookbook/01_dids/01_light_did_creation.md [hunk 2]
```diff
@@ -8,14 +8,14 @@ import TsJsBlock from '@site/src/components/TsJsBlock';
...
 
-The creation of a light DID requires the generation of some keying material for keys that are to be used for authentication, encryption, attestation, or delegation.
-For the sake of ease of use, the example snippets below show how to use a Keyring, provided also by the `@polkadot/api` library, to generate key pairs that are kept in memory and disappear at the end of the program execution, unless saved to some persistent storage.
+The creation of a light DID requires the generation of some keying material for keys that are to be used for authentication and encryption.
+For the sake of ease of use, the example snippets below show how to use keys generated with a `Keyring`, provided also by the `@polkadot/api` library, to generate key pairs that are kept in memory and disappear at the end of the program execution, unless saved to some persistent storage.
 
 :::warning
-Using the default keyring in production is highly discouraged as all the keys are kept in the memory and easily retrievable by malicious actors.
+Using the default keyring in production is highly discouraged as all the keys are kept in the memory and are more vulnerable to attacks by malicious actors.
```
Comment (rflechtner) [inline docs/develop/01_sdk/02_cookbook/01_dids/01_light_did_creation.md]: I think I asked that already, but is it really discouraged to use the polkadot keyring? What is it for then?

File: docs/develop/03_workshop/04_attester/03_ctype.md [hunk 3]
```diff
@@ -50,8 +50,6 @@ Let's have a look at these attributes.
 | `schema` > `$schema`    | The JSON schema on which the CType is based.                             |
 | `schema` > `title`      | The title of the CType.                                                  |
 | `schema` > `properties` | The properties that a claim of type in `$schema` should have.            |
-| `owner`                 | The DID of the user who created this CType.                              |
-| `hash`                  | Most important attribute, the hash is the CType's **digital footprint**. |
```
Comment (rflechtner) [inline docs/develop/03_workshop/04_attester/03_ctype.md:54]: I suggest we move the "digital footprint" idea into the description of the $id prop

File: docs/develop/03_workshop/06_attestation.md [hunk 4]
```diff
@@ -12,14 +12,13 @@ import AttestCredential from '!!raw-loader!@site/code_examples/workshop/attester
...
+- Store the attestation information on the chain;
 
 ## Attest a Credential
 
-The `attestCredential` function first initialized all libraries, as always.
-After that we can load the account and DID of the <span className="label-role attester">Attester</span>.
+The `attestCredential` function loads the account and DID of the <span className="label-role attester">Attester</span>.
 When everything is prepared, we can issue an attestation for the credential we received from the <span className="label-role claimer">Claimer</span>.
-The attestation becomes valid once it is submitted to the chain.
+The credential becomes valid once it is attested onto the chain.
```
Comment (rflechtner) [inline docs/develop/03_workshop/06_attestation.md]:
```suggestion
The credential becomes valid once an attestation exists for it on the blockchain.
```

**Review by rflechtner — APPROVED** [review]
I had a few suggestions but looks good overall!

File: docs/develop/01_sdk/02_cookbook/01_dids/01_light_did_creation.md (hunk 2 above)
Comment (ntn-x2) [reply docs/develop/01_sdk/02_cookbook/01_dids/01_light_did_creation.md]: The thing is that the keyring by itself keeps everything in memory. Production-ready keyrings like Sporran encrypt it behind a password, at least.

File: docs/develop/03_workshop/06_attestation.md (hunk 4 above)
Comment (ntn-x2) [reply docs/develop/03_workshop/06_attestation.md]: Do you think this language is better? An attestation could also be revoked, making the credential invalid.

File: docs/develop/03_workshop/04_attester/03_ctype.md [hunk 5]
```diff
@@ -44,14 +44,12 @@ For example, a very basic CType for a driver's license could look like this:
...
+| Key                     | Value                                                                                                                                                               |
+| ----------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `schema` > `$id`        | The KILT id of this CType. It is the most important property as it represents the **digital footprint** of the CType.                                               |
+| `schema` > `$schema`    | The JSON schema on which the CType is based.                                                                                                                        |
+| `schema` > `title`      | The title of the CType.                                                                                                                                             |
```
Comment (Dudleyneedham) [inline docs/develop/03_workshop/04_attester/03_ctype.md:51]: This feels broken

File: docs/develop/03_workshop/06_attestation.md (hunk 4 above)
Comment (Dudleyneedham) [reply docs/develop/03_workshop/06_attestation.md]: I think Antonio's version is better sounding.

**Review by Dudleyneedham — COMMENTED** [review]
Minor stuff

**Comment by Dudleyneedham** [comment]
Looks good!

File: docs/develop/03_workshop/06_attestation.md [hunk 6]
```diff
@@ -12,14 +12,13 @@ import AttestCredential from '!!raw-loader!@site/code_examples/workshop/attester
...
+- Store the attestation information on the chain;
 
 ## Attest a Credential
 
-The `attestCredential` function first initialized all libraries, as always.
-After that we can load the account and DID of the <span className="label-role attester">Attester</span>.
+The `attestCredential` function loads the account and DID of the <span className="label-role attester">Attester</span>.
 When everything is prepared, we can issue an attestation for the credential we received from the <span className="label-role claimer">Claimer</span>.
-The attestation becomes valid once it is submitted to the chain.
+The credential is considered to be valid from the time it is attested on chain until the time it is revoked.
```
Comment (Dudleyneedham) [inline docs/develop/03_workshop/06_attestation.md:21]: This reads better

**Review by Dudleyneedham — APPROVED** [review]
LGTM

File: docs/develop/03_workshop/04_attester/03_ctype.md (hunk 5 above)
Comment (ntn-x2) [reply docs/develop/03_workshop/04_attester/03_ctype.md:51]: It's only how it's laid out in the text format, but the generated HTML is ok.

Comment (rflechtner) [reply docs/develop/03_workshop/04_attester/03_ctype.md:51]: Damn i missed the `schema >` everywhere. I’ll open a PR

**Pull request merged**